src/js/components/Chart/doc.js

```diff
@@ -58,6 +58,9 @@ export const doc = Chart => {
     id: PropTypes.string.description(`A unique identifier for the Chart. This
       is required if more than one Chart is shown and they use color
       gradients.`),
+    dash: PropTypes.bool
+      .description(`Whether to use dashed lines for line or bar charts.`)
+      .defaultValue(false),
```

Comment:
would it be more accurate to state `undefined`? or actually set the default value of `dash` to false.

Reply:
I was following the pattern of other props, like `round`.